Clarify data retention policies for AI models (#5882)
elastic/docs-content: explore-analyze/elastic-inference/eis.md

* The applicable terms of use, uptime, and performance for each of the AI models available with EIS are each described in the applicable AI model's Provider Terms and Model Card.
* Prior to using the AI model with EIS, Customers are responsible for reviewing and agreeing to the chosen AI model's Provider Terms to understand the availability and data practices of the AI model's provider.
* After the listed end-of-life (EOL) date, the model is no longer available for {{infer}} use and requests will fail. You need to actively transition to another model before the EOL date, there is no automated migration.

> * Elastic makes models available wherever possible via providers who provide zero data retention guarantees and do not use inputs to train models. Scroll right in the above tables if you need to double-check the status of a specific model.

Author: ```suggestion
* Elastic makes models available wherever possible using providers who provide zero data retention guarantees and do not use inputs to train models. Scroll right in the above tables if you need to double-check the status of a specific model.
```

> * Elastic makes every effort to use third party providers who do not use inputs to train models, and do not retain any data (zero data retention). Scroll right in the above tables if you need to double-check the status of a specific model.

Reviewer: ```suggestion
* Elastic makes every effort to use third party providers who do not use inputs to train models, and do not retain any data (zero data retention). Browse the tables on this page to double-check the status of a specific model.
```
